pkg/asset/installconfig/ovirt/ovirt.go

```diff
@@ -92,5 +92,19 @@ func Platform() (*ovirt.Platform, error) {
 		return nil, errors.Wrap(err, "failed UserInput")
 	}
 
+	err = survey.Ask([]*survey.Question{
+		{
+			Prompt: &survey.Confirm{
+				Message: "Use thick disk provisioning for master nodes?",
+				Help:    "Thick provisioning improves master node performance, but makes the installation process slower and will consume the entire disk up-front on the oVirt cluster.",
```

Comment:
thinking a bit more about it, I wouldn't want to claim that it's slower. Some aspects are slower for sure, the initial master VMs creation. But it's most likely not that significant considering the overall deployment time. And on the other hand the operations are supposedly a tiny bit faster so it easily could be that the rest of the procedure offsets for it. 
I would rephrase and leave out "makes the installation process slower and"